calculators/committee_contribution_list_calculator.rb

```diff
@@ -1,14 +1,24 @@
 class CommitteeContributionListCalculator
   def initialize(candidates: [], ballot_measures: [], committees: [])
     @committees = committees
+    @candidates = candidates
   end
 
   def fetch
     results = ActiveRecord::Base.connection.execute(<<-SQL)
-      SELECT "Filer_ID", "Tran_Amt1", "Tran_Date", "Tran_NamF", "Tran_NamL",
+      SELECT all_contributions."Filer_ID", "Tran_Amt1", "Tran_Date", "Tran_NamF", "Tran_NamL",
         "Tran_Zip4", "Tran_Occ", "Tran_Emp"
-      FROM combined_contributions
-      WHERE "Filer_ID" IN (#{filer_ids})
+      FROM all_contributions
+      JOIN (
+        SELECT DISTINCT "Filer_ID", "Start_Date", "End_Date" FROM oakland_committees
+        WHERE NOT EXISTS (SELECT * FROM oakland_candidates
+                          WHERE "FPPC"::varchar = "Filer_ID")
+        UNION ALL
+        SELECT "FPPC"::varchar AS "Filer_ID", "Start_Date", "End_Date" FROM oakland_candidates
+      ) committees
```

Comment:
yikes, can we use `WITH` expressions to give inner queries names, e.g.

```sql
WITH (
  SELECT DISTINCT "Filer_ID", "Start_Date", "End_Date" FROM oakland_committees
  WHERE NOT EXISTS (SELECT * FROM oakland_candidates WHERE "FPPC"::varchar = "Filer_ID")
  UNION ALL
  SELECT "FPPC"::varchar AS "Filer_ID", "Start_Date", "End_Date" FROM oakland_candidates
) as committees_and_candidates
```

Reply:
I spose.  The inner query does have a name "committees"